packages/std/src/ado_contract/query.rs

```diff
@@ -108,4 +110,12 @@ impl<'a> ADOContract<'a> {
             version: contract_version.version,
         })
     }
+
+    #[inline]
+    pub fn query_ado_base_version(&self) -> Result<ADOBaseVersionResponse, ContractError> {
+        let ado_base_version: &str = env!("CARGO_PKG_VERSION");
```

Comment:
Does this use the correct version? Have you tried manipulating it in a test to verify?

Reply:
Yes I changed `andromeda-std`'s version locally to ensure that we're fetching it instead of the contract's version. 
But it feels like there should be a better way of doing this.